README.md

```diff
@@ -0,0 +1,46 @@
+# API Practice
+Check out this project [here](https://twitter.com/_gg_bot). This is a link to my twitter that is 
```

Comment:
There is no need for newlines in a README.

Reply:
This has been fixed